addons/mail/static/src/core/common/attachment_list.xml

```diff
@@ -23,8 +23,15 @@
                      role="menuitem"
                 >
                     <!-- display of image attachments-->
+                    <Gif
+                        t-if="attachment.extension === 'gif'"
```

Comment:
Not directly related to this PR but `extension` is not a reliable way to identify files, we should be getting rid of this field completely and always use mimetype instead.